templates/features/addons.html

```diff
@@ -0,0 +1,18 @@
+<section class="p-section--shallow">
+  <hr>
+  <div class="row">
+    <div class="col-6">
+      <div class="p-section--shallow">
+        <h2>Better user experience with addons</h2>
+      </div>
+      <div class="p-section--shallow">
+        <p>To keep MicroK8s light and flexible, the standard install only has the components needed to deploy Kubernetes. Depending on the use case, there are plenty of additional Kubernetes services or operators that would be nice to have.</p>
+        <p>MicroK8s addresses this with addons - extra services that can easily be added to MicroK8s. You can enable or disable addons at any time. Most are pre-configured to 'just work' without any further set up.</p>
```

Comment:
```suggestion
        <p>MicroK8s addresses this with addons &mdash; extra services that can easily be added to MicroK8s. You can enable or disable addons at any time. Most are pre-configured to 'just work' without any further set up.</p>
```
Encoded mdashes are much nicer.